.github/workflows/lean4checker.yml

```diff
@@ -78,6 +78,15 @@ jobs:
           cd ..
           lake env lean4checker/.lake/build/bin/lean4checker
 
+      - id: ci_url
+        run: |
+          url=https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          job_no="$(curl -s "${url}" |
+            # extract the CI run number from the unique line that matches `  href=".../job/[job number]"`
+            awk -v url="${url}" -F'/' '/^ *href/ {gsub(/"/, "", $NF); printf("summary<<EOF\n%s/job/%s\nEOF", url, $NF)}')"
+          printf '%s\n' "${job_no}"
```

Comment:
Is this duplication needed?

Reply:
Shortened it, thanks!  This was for testing.